deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
+WHICH=$(which docker)
+SCRIPTPATH=$(pwd -P)
+FILE1="docker-compose-base.yaml"
```

Comment:
Suggest using more descriptive variable names.  I don't know the difference between `FILE1` and `FILE2` without having to consult this code snippet.

Reply:
I had it generic on purpose, but ok. I changed it to a more descriptive name if that is preferred.